languages/ro-RO.js

```diff
@@ -69,15 +69,16 @@ module.exports = class extends Language {
 				'Majoritatea codului este modularizat, ceea ce permite dezvoltatorilor să editeze Klasa după necesitățile lor.',
 				'',
 				'Unele din caracteristici Klasa sunt:',
-				'• Lansare rapidă cu ajutorul la ES7 (Async/Await)',
-				'• Setări pentru fiecare guild in parte care poate fi extins la propriul tău cod',
-				'• Un sistem de comenzi care poate fi modificat cu utilizarea argumentelor fiind automată, reîncărcarea rapidă și descărcarea noilor module',
-				'• "Monitoare" care pot monitoriza orice mesaj si a acționa asupra lor, ca un eveniment de mesaj normal (Filtru pentru injurături, Protecție de Spam, etc)',
-				'• "Inhibitoare" care interzic rularea comenzilor în baza parametrilor (Permisiuni, Liste negre, etc)',
-				'• "Providere" care permit conectarea la orice bază de date externă dorești.',
-				'• "Finalizere" care rulează pe mesaje după ce o comandă a rulat cu succes.',
-				'• "Extendabile", cod care acționează pasiv. Ele adaugă proprietăți și metode la clasele existente din Discord.js.',
-				'• "Limbi", care permit localizare botului tău.',
+				'• 🐇💨 Lansare rapidă cu ajutorul la ES2017 (`async`/`await`)',
+				'• 🎚🎛 Setări pentru fiecare guild in parte care poate fi extins la propriul tău cod',
+				'• 💬 Un sistem de comenzi care poate fi modificat cu utilizarea argumentelor fiind automată, reîncărcarea rapidă și descărcarea noilor module',
+				'• 👀 "Monitoare" care pot monitoriza orice mesaj si a acționa asupra lor, ca un eveniment de mesaj normal (Filtru pentru injurături, Protecție de Spam, etc)',
+				'• ⛔ "Inhibitoare" care interzic rularea comenzilor în baza parametrilor (Permisiuni, Liste negre, etc)',
+				'• 🗄 "Providere" care permit conectarea la orice bază de date externă dorești.',
+				'• ✅ "Finalizere" care rulează pe mesaje după ce o comandă a rulat cu succes.',
+				'• ➕ "Extendabile", cod care acționează pasiv. Ele adaugă proprietăți și metode la clasele existente din Discord.js.',
```

Comment:
`"Extendabile", care adaugă pasiv metode, getteri sau setteri,  sau chiar proprietăți statice către clase care există deja in Discord.js sau Klasa`